master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml

```diff
@@ -67,14 +67,15 @@ spec:
         # a failure.  This annotation works in tandem with the toleration below.
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
-      # Only run this pod on the master.
+      # Toleration allows the pod to run on master 
```

Comment:
Could you move the comment to just above the `key: node-role.kubernetes.io/master` so it doesn't sound like the CriticalAddonsOnly one is for the same thing?
Also please add this in the same place down below for the calico-node DaemonSet as it is appropriate there too.